main/influxdb_watcher.go

```diff
@@ -32,7 +32,7 @@ import (
 )
 
 type watcherConfiguration struct {
-	InfluxdbAddress    string `env:"INFLUXDB_ADDRESS" envDefault:"http://localhost:8086"`
+	InfluxdbAddress    string `env:"INFLUXDB_ADDRESS" envDefault:"http://influxdb:8086"`
```

Comment:
I think it should stay `localhost` by default and if user want then he could change it to any docker link.